conf.py

```diff
@@ -42,8 +42,17 @@
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
     "sphinx_version_ref",
+   "sphinxcontrib.spelling",
```

Comment:
nitpick: can you make sure that this line is inline with the others please and thank you!